index.html

```diff
@@ -41,8 +41,124 @@ <h3 class="link-1"> <a href="#"> LET'S CONNECT</a></h3>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <button  class="tag">html</button>
+                    <button  class="tag">css</button>
+                    <button  class="tag">javascript</button>
+                </div>
+                <button type="submit" class="normal-button">See Project</button>
+            </div>
+
+                   <div class="card">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio(4).png" alt="">
+                <h1 class = "title">E-Commerce</h1>
+                <ul class="frame">
+                    <li class="detail">CANOPY</li>
+                    <li class="detail">BACK END DEV</li>       
+                    <li class="detail">2015</li>       
+                </ul>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <button  class="tag">html</button>
+                    <button  class="tag">css</button>
+                    <button  class="tag">javascript</button>
+                </div>
+                <button type ="submit" class="normal-button">See Project</button>
+            </div>
+
+                   <div class="card">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio(5).png" alt="">
+                <h1 class = "title">BOOK.com</h1>
+                <ul class="frame">
+                    <li class="detail">CANOPY</li>
+                    <li class="detail">BACK END DEV</li>       
+                    <li class="detail">2015</li>       
+                </ul>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <button  class="tag">html</button>
+                    <button  class="tag">css</button>
+                    <button  class="tag">javascript</button>
+                </div>
+                <button type="submit" class="normal-button">See Project</button>
+            </div>    
+        </section> 
+ <section id="about_me">
+            <div class="data-inf">
+              <h1>About<br> Myself</h1>
+              <p id="presentation">Hello I’m a data Scientist and software developer! I can help you build a product ,
+                  feature or website Look through some of my work and experience!
+                   If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <div id="profile_link">
+                <h2>LET’S CONNECT</h2>
+                <ul>
+                    <li><img src="./image/github.png" alt="img"></li>
+                    <li><img src="./image/Linkedin_icon.png" alt="img"></li>
+                    <li><img src="./image/doits.png" alt="img"></li>
+                </ul>
```

Comment:
[OPTIONAL] Same here, please try to make the description to be on the left side and the languages on the right. You can use Flexbox to achieve this.
![image](https://user-images.githubusercontent.com/38088818/166099456-39c10fa1-99a5-466a-89e4-ab566c597d37.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/166099497-75c315fa-1acb-4998-b3b9-0a2f3e60598c.png)